src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -151,7 +151,7 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
         const currentFileVersionId = getProp(file, 'file_version.id');
         const fileVersionId = getProp(match, 'params.fileVersionId');
 
-        if (fileVersionId && annotationId && fileVersionId !== currentFileVersionId) {
+        if (fileVersionId && fileVersionId !== currentFileVersionId) {
```

Comment:
This method isn't really specific to annotations anymore, since it's always called on mount and doesn't check for annotation id. Should we name it `redirectInitialPath` or something?

Reply:
I think I'll leave it for now, it still relies on the `getAnnotationsMatchPath` to check for and set back an annotations location path